website/docs/components/form/segmented-group/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Segmented Group
+description: Combines one or more input fields and actions to handle complex filtering and data collection.
+caption: Combines one or more input fields and actions to handle complex filtering and data collection.
+status: experimental
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=30150%3A45701&t=akpWrhSdwygOH6md-1
+previewImage: assets/illustrations/components/form/segmented-group.jpg
+navigation:
+  keywords: [ 'filter', 'connected', 'fields', 'set' ]
```

Comment:
do we want to add more keywords here, related to what it is (eg. `group`, `segment`) and contains (eg. `button`, `dropdown`, `input`, `select`)?

Reply:
Added these, the only thing that gives me a little pause is including the items it can contain (might muddy up the search a little bit). I did a bit of testing, it seems reasonable but if a consumer doesn't have any concept of a what a `Segmented Group` is they likely won't understand the correlation right off the bat. This probably isn't a problem though.